types/classnames/classnames-tests.ts

```diff
@@ -1,5 +1,5 @@
 import classNames = require('classnames');
-import * as classNames2 from 'classnames';
+import classNames2 from 'classnames';
```

Comment:
Why did this need to change? Did the supported import style change?

Reply:
Was not intentional -- fixed.